Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -252,19 +252,6 @@ class BrowserViewController: UIViewController,
         overlayManager.switchTab(shouldCancelLoading: true)
     }
 
-    @objc
-    func openTabNotification(notification: Notification) {
```

Comment:
Further clean up: The `OpenTabNotificationObject` file can be removed as well as the NotificationConstant `public static let OpenTabNotification`

Reply:
Thanks for catching 👍 , forgot to remove those. Should be fixed now ✅